src/pudl/transform/classes.py

```diff
@@ -299,7 +320,11 @@ def mapping(self) -> dict[str, str]:
 
 
 def categorize_strings(col: pd.Series, params: StringCategories) -> pd.Series:
-    """Impose a controlled vocabulary on freeform string column."""
+    """Impose a controlled vocabulary on a freeform string column.
```

Comment:
I think controlled vocabulary is correct here and common for when you're imposing a constrained set of words to categorize things clearly, rather than allowing every possible natural variation. All the different subtypes of coal and all their misspellings just become `coal`. Categorizing freeform strings into a restricted set of allowed canonical values (of fuel type, plant type, etc.) is imposing a controlled vocabulary isn't it? We've described it this way for a long time.